Client/Application/AppDelegate.swift

```diff
@@ -127,11 +128,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         navigationController.delegate = self
         navigationController.isNavigationBarHidden = true
 
-        if AppConstants.MOZ_STATUS_BAR_NOTIFICATION {
-            rootViewController = NotificationRootViewController(rootViewController: navigationController)
-        } else {
+//        if AppConstants.MOZ_STATUS_BAR_NOTIFICATION {
+//            rootViewController = NotificationRootViewController(rootViewController: navigationController)
+//        } else {
             rootViewController = navigationController
-        }
+//        }
```

Comment:
This is a question or @sleroux -- I think he said something about this code isn't really needed anymore since some work that he did is not used anymore? Plus, if we don't have this ensured then deeplinks can't work